lib/ssm_config.rb

```diff
@@ -1,12 +1,19 @@
 class SsmConfig
   VERSION = '0.1.1'.freeze
   CONFIG_PATH = 'config'.freeze
-
+  TABLE_NAME = 'ssm_config_records'.freeze
+  ACTIVE_RECORD_MODEL = 'SsmConfigRecord'.freeze
   class << self
     def method_missing(meth, *args, &block)
       config_file = Rails.root.join(CONFIG_PATH, "#{meth}.yml")
 
-      if File.exist?(config_file)
+      if ActiveRecord::Base.connection.table_exists? TABLE_NAME
+        if ACTIVE_RECORD_MODEL.constantize.exists?(:file => meth.to_s)
```

Comment:
Should we also add a `ACTIVE_RECORD_MODEL.constantize.defined?` to the if statement? It we don't have the model, does it call `exists?` on nil?

Reply:
I guess the assumption was that if `ssm_config_records` exists then the model `SsmConfigRecord` exists, but I can add a quick change just in case